renderer/src/components/Content.js

```diff
@@ -13,15 +13,20 @@ export default class Content extends Component {
     const { content } = this.props;
 
     return (
-      <Tabs defaultActiveKey="1" onChange={this.callback}>
-      {
-        content.diff.map((diffItem, key) => (
-          <TabPane tab={diffItem.file} key={key}>
-            <ContentItem renderItem={diffItem.explain} />
-          </TabPane>
-        ))
-      }
-      </Tabs>
+      <div style={{ width: '70%', marginLeft: '20px' }}>
+        <Tabs 
+          defaultActiveKey="1" 
+          onChange={this.callback}
+        >
+        {
+          content.diff.map((diffItem, key) => (
+            <TabPane tab={diffItem.file} key={key}>
+              <ContentItem renderItem={diffItem.explain} />
+            </TabPane>
+          ))
+        }
+        </Tabs>
+      </div>
     );
   }
 }
```

Comment:
Later I'll add `.editorconfig` to enforce new line at the end of file.

Reply:
Oh, a good idea, I think

Reply:
I'll add it